<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaRanking</class>
 <widget class="QMainWindow" name="VentanaRanking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 74)</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(80, 255, 248)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30pt &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Ranking de puntajes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_tercero">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 243, 26);
font: 75 18pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_segundo">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 243, 26);
font: 75 18pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_primero">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 243, 26);
font: 75 18pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="boton_volver_ranking">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>460</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 250, 106)</string>
      </property>
      <property name="text">
       <string>volver</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cuarto">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 243, 26);
font: 75 18pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_quinto">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>390</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 243, 26);
font: 75 18pt &quot;Marker Felt&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
